core/src/main/scala/org/apache/spark/BarrierCoordinator.scala

```diff
@@ -187,6 +191,12 @@ private[spark] class BarrierCoordinator(
       requesters.clear()
       cancelTimerTask()
     }
+
+    // Check for clearing internal data, visible for test only.
+    private[spark] def cleanCheck(): Boolean = requesters.isEmpty && timerTask == null
```

Comment:
nit: `cleanCheck()` -> `isInternalStateClear()`

Reply:
Thanks, done in fd4d150.